### ISSUE TITLE
Controller Support

### ISSUE THREAD

**Issue by SubstanceSix** [SoapboxRaceWorld/GameLauncher_NFSW#11]
- **Launcher Version: 2.0.2.8
- **Operating system: Win10
- **Antivirus/Firewall (if any): Windows Defender
- **Wine/Mono Version (if any): ?

Hi, I used to be able to run a DS4 with the game by running the original launcher through Steam and by having DS4Windows open at the same time. 

Now that the original launcher stopped connecting with the servers (OUTDATEDSERVERLIST) I downloaded this GameLauncher, but I see that while it is active it completely blocks any function from the controller, be it connected via DS4Windows or Steam drivers. I notice this from functions such as the touchpad to move the mouse cursor or keyboard key bindings stop working.

When I exit the game or click outside the launcher and minimize it the controller is working again.

**Comment by ghost** [comment]
Verified to work for me with that setting:
![](https://files.metonator.cloud/up/DS4Windows_uNZB6GHpsb.a36b4f33.png)

**Comment by SubstanceSix** [comment]
Thanks for the input, do you run it while having Steam open at all? And can you use it both wirelessly and wired? 
My DS4W is a little bit different than yours by the way, I don't have "Run as" option and "Uninstall VBus Driver there, it is the latest version 1.6.12. Other than that, I don't see an option there that would affect functionality apart from Xinput ports and UDP Server which are the same as yours :/ .

Thank you

![image](https://user-images.githubusercontent.com/48032332/53450948-7d27df00-3a15-11e9-89f0-0d62df010f47.png)

![image](https://user-images.githubusercontent.com/48032332/53450967-8749dd80-3a15-11e9-8498-e1a915b05948.png)

**Comment by ghost** [comment]
I mostly use it wireless with steam enabled, the only thing i must make sure is not turned on is in fact GeForce Experience Overlay.

**Comment by SubstanceSix** [comment]
I have GeForce Experience completely disabled. I did restore basic windows functions such as the ones I mentioned earlier, cursor movement from touchpad input and keyboard bindings, by running DS4W as administrator, so we know that the controller doesn't somehow get completely blocked now. But the game won't seem to recognise the controller as a controller to play with it.

I can't think of anything else atm, I tried hooking the DS4 with either DS4W or Steam or both, running the launcher alone or through Steam. If only there was a way to use the old launcher
![image](https://user-images.githubusercontent.com/48032332/53454852-7ce11100-3a20-11e9-80cb-980f6f735221.png).
